subhub/api/payments.py

```diff
@@ -101,7 +103,7 @@ def subscription_status(uid) -> FlaskResponse:
     subscriptions = stripe.Subscription.list(
         customer=items.custId, limit=100, status="all"
     )
-    if subscriptions is None:
+    if len(subscriptions) == 0:
```

Comment:
I'm used to seeing this check done as:
```python
if not subscriptions:
```

Was there some other reason not to rely on typical [Python truth value testing](https://docs.python.org/3/library/stdtypes.html#truth-value-testing)?

Reply:
Oversight, I corrected it.